### ISSUE TITLE
Kustomize released assets naming scheme changed again

### ISSUE THREAD

**Issue by looztra** [kubernetes-sigs/kustomize#1691]
up to v3.2.0 => kustomize_${version}_linux_amd64
from v3.2.1 to v3.2.3 => kustomize_kustomize.v${version}_linux_amd64
v3.3.0 => kustomize_v3.3.0_linux_amd64.tar.gz

This is hard to follow when you want to script kustomize installation or maintain something like the asdf plugin for kustomize :(

**Comment by vreon** [comment]
Related:
- #1602 
- Banno/asdf-kustomize#2 (thanks for this @looztra)
